### ISSUE TITLE
[FEATURE] MotionNCP 2.5 blocks height

### ISSUE THREAD

**Issue by yorik100** [CCBlueX/LiquidBounce1.8-Issues#3589]
**Describe your feature request.**
NCP step that can go 2.5 blocks high (as seen in Sigma)

**Additional context**

**Comment by kawaiinekololis** [comment]
yeah but the ncp motion step in liquidbounce can't go up to 2.5 block so ?

**Comment by yorik100** [comment]
> 
> 
> yeah but the ncp motion step in liquidbounce can't go up to 2.5 block so ?

That's why it's a request

**Comment by kawaiinekololis** [comment]
Yes and I probably never going to add it.
